e2e/test/scenarios/dashboard/tabs.cy.spec.js

```diff
@@ -416,16 +417,22 @@ describe("scenarios > dashboard > tabs", () => {
     sidebar().within(() => {
       cy.findByText("Orders, Count").click();
     });
+
+    cy.wait("@cardQuery");
+
     saveDashboard();
 
     cy.wait("@saveDashboardCards").then(({ response }) => {
       cy.wrap(response.body.dashcards[1].id).as("secondTabDashcardId");
     });
 
+    // it's possible to have two requests firing (but first one is canceled before running second)
```

Comment:
I think our specs should not adapt to different environemnts, why not to wait until requests are resolved? https://github.com/metabase/metabase/pull/39783#top